packages/terra-doc-template/src/DocTemplate.jsx

```diff
@@ -127,6 +132,31 @@ const DocTemplate = ({
           />
         ))}
       </div>
+
+      {themeVariables && (
+        <div className={cx('doc-card')}>
+          <div className="markdown-body">
+            <h2>Theme Variables (CSS Custom Properties)</h2>
+            <table>
+              <thead>
+                <tr>
+                  <th>Property Name</th>
+                  <th>Property Value</th>
+                </tr>
+              </thead>
+              <tbody>
+                {Object.keys(themeVariables).map(key => (
+                  <tr>
+                    <td>{key}</td>
+                    <td>{themeVariables[key]}</td>
```

Comment:
not a big deal, but for readability can we encase theme variable values in back-ticks?